src/runtime_src/core/edge/sw_em/generic_pcie_hal2/shim.h

```diff
@@ -244,6 +245,7 @@ namespace xclcpuemhal2 {
       exec_core* mCore;
       SWScheduler* mSWSch;
       bool mIsKdsSwEmu;
+      xclBin *mHeader;
```

Comment:
not needed, use mCoreDevice

Reply:
Removed xclbin *mHeader and used mCoreDevice to get IP_LAYOUT section.